bundles/AdminBundle/public/js/pimcore/asset/tree.js

```diff
@@ -758,7 +758,10 @@
              var advancedMenuItems = [];
              var user = pimcore.globalmanager.get("user");
  
-             if (record.data.permissions.create && !record.data.locked && perspectiveCfg.inTreeContextMenu("asset.searchAndMove")) {
+             if (record.data.permissions.create &&
+                 !record.data.locked &&
+                 perspectiveCfg.inTreeContextMenu("asset.searchAndMove") &&
+                 pimcore.globalmanager.get('searchImplementationRegistry').hasImplementation()) {
```

Comment:
we could add a `pimcore.helpers.hasSearchImplementation()` and put the `pimcore.globalmanager.get('searchImplementationRegistry').hasImplementation())`there ... would make the code a bit more readable?

Reply:
done 2021a6015e96942d11b9ff29e0c4167d6a952003